apps/cloudbroker/base/libcloud__libvirt/specs/model.spec

```diff
@@ -9,6 +9,7 @@
     prop:size int,, size in MByte
     prop:type str,, dot separated list of independant terms known terms are: tar;gz;sso e.g. sso dump inn tar.gz format would be sso.tar.gz  (always in lcas)
     prop:extra str,, extra data linked to the image
+    prop:bootType str,, image boot type e.g bios, uefi
```

Comment:
this should be added on the image object of cloudbroker
Please delete the image, size, resourceprovider and node object from this file its not used anymore